### PR TITLE
Welcome to Keda - Dig Security

### PR DESCRIPTION
**Pull request by Shahard2**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -369,6 +369,10 @@ logo = "baseten.png"
 url = "https://www.bolddesk.com/"
 logo = "bolddesk.svg"
 
+[[params.users]]
```
Comment (zroubalik) [inline config.toml:372]: Welcome!

Could you please sort this alphabetically.

Comment (Shahard2) [reply config.toml]: Sure @zroubalik can you check now?

**Review by zroubalik — APPROVED** [review]

**Comment by zroubalik** [comment]
🚀

**Comment by zroubalik** [comment]
Thanks!

**Comment by tomkerkhove** [comment]
Lovely, thanks for opening the PR!

**Comment by Shahard2** [comment]
> Lovely, thanks for opening the PR!

Thank you mate! :)

**Comment by JorTurFer** [comment]
Welcome!!!! ❤️

**Comment by Shahard2** [comment]
> Welcome!!!! ❤️

Thank you mate!!!!

**Pull request merged**